components/esm/loadcrea.hpp

```diff
@@ -28,20 +28,21 @@ struct Creature
     // Default is 0x48?
     enum Flags
     {
-        // Movement types
-        Bipedal       = 0x001,
-        Swims       = 0x010,
-        Flies       = 0x020, // Don't know what happens if several
-        Walks       = 0x040, // of these are set
-
-        Respawn     = 0x002,
-        Weapon      = 0x004, // Has weapon and shield
-        None        = 0x008, // ?? This flag appears set for every creature in Morrowind.esm
-        Essential   = 0x080,
-
-        // Blood types
-        Skeleton    = 0x400,
-        Metal       = 0x800
+        Bipedal       = 0x0001,
+        Respawn       = 0x0002,
+        Weapon        = 0x0004, // Has weapon and shield
```

Comment:
I think you can also convert comments to doc comments

Reply:
Not a priority.